src/module/vue/components/resource-meter/pc-condition-meters.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="condition-meters flexcol">
+    <ConditionMeterSlider
+      v-for="resource in ['Health', 'Spirit', 'Supply']"
+      sliderStyle="vertical"
```

Comment:
Opening this in Code I get a red squiggly over the whole thing. This fixes it:

```suggestion
      v-for="resource in ['Health', 'Spirit', 'Supply']"
      :key="resource"
      sliderStyle="vertical"
```